src/hotspot/share/adlc/adlparse.cpp

```diff
@@ -476,7 +478,7 @@ void ADLParser::oper_parse(void) {
       parse_err(SYNERR, "Operands do not specify an effect\n");
     }
     else if (!strcmp(ident, "expand"))    {
-      parse_err(SYNERR, "Operands do not specify an expand\n");
+      oper_expand_parse(oper);
```

Comment:
Should that be a function called like `oper->_expand = expand_parse();`, to align with the other *_parse functions.